CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.15]
+---------
```

Comment:
I guess it should be limited to what is happening in this PR. e.g feat: Adding new fields `enable_learner_portal_sidebar_message` and `learner_portal_sidebar_content` in EnterpriseCustomer.